renderdoc/data/glsl_shaders.cpp

```diff
@@ -30,7 +30,10 @@ void GenerateGLSLShader(std::vector<std::string> &sources, ShaderType type,
                         bool uniforms)
 {
   sources.resize(4);
-  sources[0] = StringFormat::Fmt("#version %d core\n", version);
+  sources[0] = StringFormat::Fmt("#version %d %s\n", version, type == eShaderGLSLES ? "es" : "core");
+
+  if(type == eShaderGLSLES)
+    sources[0] += "#define OPENGL_ES 1\n";
```

Comment:
Can we not use the `GL_ES` predefined macro that already exists in GL ES's shading language? If you prefer you could do

```
#ifdef GL_ES
#define OPENGL_ES 1
#endif
```

in debuguniforms.h.